amundsen_application/static/js/pages/DashboardPage/index.tsx

```diff
@@ -316,6 +327,17 @@ export class DashboardPage extends React.Component<
   }
 }
 
+export function searchDashboardGroup(
```

Comment:
nit: don't need to export if this function is only referenced in this file.

Reply:
cool, pr updated